joern-cli/frontends/javasrc2cpg/src/main/scala/io/joern/javasrc2cpg/util/Delombok.scala

```diff
@@ -0,0 +1,54 @@
+package io.joern.javasrc2cpg.util
+
+import better.files.File
+import io.joern.x2cpg.utils.ExternalCommand
+import org.slf4j.LoggerFactory
+
+import java.nio.file.Paths
+import scala.util.{Failure, Success, Try}
+
+object Delombok {
+  private val logger = LoggerFactory.getLogger(this.getClass)
+
+  private val javaPath: String = {
+    sys.env
+      .get("JAVASRC2CPG_JAVA_HOME")
```

Comment:
Three things about this env variable:
1. It is not really descriptive
2. I suggest, that however we will name this variable, it will also be used to define the java version for the JavaParser type resolution. After all it just makes sense to have that the same as the lombok target version.
3. I suggest we make the env variable a parameter and call it "--analysis-java-home".